packages/messaging/src/window.ts

```diff
@@ -16,8 +16,14 @@ export interface WindowMessagingConfig extends NamespaceMessagingConfig {}
  *
  * > See <https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#targetorigin> for more
  * details.
+ *
+ * message is posted on window which can as per your need like
+ * parent window in iframe -> window.parent
+ * iframe content window -> iframe.contentWindow
+ * opener	original window -> window.opener
+ * by default global window is used to send mesage
  */
-export type WindowSendMessageArgs = [targetOrigin?: string];
+export type WindowSendMessageArgs = [targetOrigin?: string, targetWindow?: Window];
```

Comment:
can have kept targetWindow first and then targetOrigin but don't this to be released as major version